arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -651,6 +651,56 @@
         }
       }
     },
+    "ClusterPropertiesUpdateParameters": {
+      "description": "The cluster resource properties",
+      "properties": {
+        "reliabilityLevel": {
+          "type": "string",
+          "enum": [
+            "Invalid",
+            "Bronze",
+            "Silver",
+            "Gold",
+            "Platinum"
+          ]
+        },
+        "upgradeMode": {
+          "type": "string",
+          "enum": [
+            "Default",
+            "Automatic",
+            "Manual"
+          ]
+        },
+        "clusterCodeVersion": {
+          "type": "string"
```

Comment:
1. Can we have the description for `clusterCodeVersion` property? I mean a meaningful description like "the version of fabric runtime hosted in the cluster" (this may not be correct description, just put so to emphasize that by reading description user gets clear picture of this property)

2. Confirming can the user change this property (via PATCH)? Asking because I don't see such a property in Cluster model used for PUT.